plugins/Bookmark.jsx

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright 2020-2021 Sourcepole AG
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import {connect} from 'react-redux';
+import PropTypes from 'prop-types';
+import ConfigUtils from '../utils/ConfigUtils';
+import BookmarkWindow from '../components/BookmarkWindow';
+ 
+class Bookmark extends React.Component {
```

Comment:
Is there a reason for creating the wrapping plugin for the component instead of just having the entire component as the plugin in one single file? Also, what do you think about using a sidebar (like the Share plugin) rather than a window?

Reply:
@manisandro I think I have created the plugin to be able to enable/disable it in `config.json` file. But I realise it may be useless and I should have the component as the plugin ?

I tried to use a sidebar and a window for my tests. I just preferred the window because it is resizeable and dockable but it can be discussed :)

Reply:
Yes you can spare yourself the boilerplate and just expose the component itself as a plugin.

Personally I'd go with a sidebar for consistency with the other functionalities, but your call.

Reply:
@manisandro I did the changes to use component directly as a plugin.

Also, I think you were right about the SideBar, it is more consistent for the user to get all the functionalities at the same place :+1: